fundingmanager.go

```diff
@@ -263,6 +282,14 @@ type fundingConfig struct {
 	// discovered short channel ID of a formerly pending channel to outside
 	// sub-systems.
 	ReportShortChanID func(wire.OutPoint, lnwire.ShortChannelID) error
+
+	// ZombieSweeperInterval is the periodic time
```

Comment:
nit: lines should be wrapped at 80 characters

Reply:
Fixed